### PR TITLE
Feat/page headline component

### PR DESCRIPTION
**Pull request by trinegl**

### CODE REVIEW

File: _includes/page-headline.liquid [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<h1 
+  {% for attribute in data.headline.htmlAttributes %}
+    {{ attribute[0] }}={{ attribute[1] }}
+  {% endfor %}
+>
```
Comment (Wandkleister) [inline _includes/page-headline.liquid:5]: Ok that's a little overengineered :-D 
The classes of this element will need to be the same, everywhere we use it, to share a certain styling - so I would suggest this:

```suggestion
<h1 {% if classModifier %} 
class="headline-1 {{data.headline.classModifier}}" 
{% else %} class="headline-1" {% endif %}
>
```

With that we stick to a general class named "headline-1". If we would need something special on this component we add a class-modifier, like "is-red".

File: _includes/page-headline.liquid [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      {% endif %}
+    >
+      {{ headlineElement.text }}
+    </span>
+
+    {% if headlineElement.lineBreakAfter %}
+      <br>
+    {% endif %}
+
+  {% endfor %}
```
Comment (Wandkleister) [inline _includes/page-headline.liquid:22]: I would also make this easier. Right now we assume, we will have a maximum of 2 lines for a headline component. So lets stick to that:

```suggestion
{{ data.headlineElement.firstline }}
{% if headlineElement.secondline %}
<br>
{{ data.headlineElement.secondline }}
{% endif %}
```

No need to use a for loop here in my point of view.

File: styles/includes/_page-headline.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.data-policy {
```
Comment (Wandkleister) [inline styles/includes/_page-headline.scss:1]: The class naming should be fixed, if you stick to general class naming from my first comment.
The class `.data-policy` shouldn't be in a style-file of a generic component

**Review by Wandkleister — CHANGES_REQUESTED** [review]

File: _includes/page-headline.liquid (hunk 2 above)
Comment (trinegl) [reply _includes/page-headline.liquid:22]: Yes I see your point. The reason I made it like this is that based on the landing page design and the (not yet finalized) design iterations in Figma we would probably need to also style parts of single words (eg italic or regular font like "brains that build" on the landing page) and not just add line breaks - this is of course in the future, but that's why I added this. Not sure if there's a better way or if this should be simplified for now?

Comment (Wandkleister) [reply _includes/page-headline.liquid:22]: Ok fair point. But I would assume that normally we have pretty standard headlines. 
If crazy word-stylings come around, we could add another component for those words and add a new if-statement in the headline template. This makes it more readable and easier to handle.
Of course your approach gives us the flexibility to change everything, but I would question if we really need that?

Comment (trinegl) [reply _includes/page-headline.liquid:24]: Yeah I honestly don't know if we'll need it, I just assumed based on what I've seen from design so far, but we can also wait and then make a new component when needed? If so I'll apply the changes you suggested :)

Comment (trinegl) [reply _includes/page-headline.liquid:24]: Or actually, maybe I could just move the flexible part into a new component, and if we won't need it we can just delete that component?

Comment (trinegl) [reply _includes/page-headline.liquid]: Removed the flexible part, we can always add a new component later :)

**Pull request merged**